addons/account/wizard/account_move_send_batch_wizard.py

```diff
@@ -31,11 +33,23 @@ def default_get(self, fields_list):
     @api.depends('move_ids')
     def _compute_summary_data(self):
         sending_methods = dict(self.env['res.partner']._fields['invoice_sending_method'].selection)
+        extra_edis = self._get_all_extra_edis()
+
         for wizard in self:
-            wizard.summary_data = {
-                sending_method: {'count': len(moves), 'label': sending_methods[sending_method]}
-                for sending_method, moves in wizard.move_ids.grouped(self._get_default_sending_method).items()
-            }
+            sending_method_counter = Counter()
+            edi_counter = Counter()
+
+            for move in wizard.move_ids:
+                sending_method_counter[self._get_default_sending_method(move)] += 1
+                edi_counter += Counter([edi for edi in self._get_default_extra_edis(move)])
+
+            summary_data = dict()
+            for sending_method, sending_method_count in sending_method_counter.items():
+                summary_data[sending_method] = {'count': sending_method_count, 'label': sending_methods[sending_method]}
+            for edi, edi_count in edi_counter.items():
```

Comment:
Shouldn't we put extra/governmental EDI first like in the single wizard ? :thinking: 
Otherwise LGTM :+1:

Reply:
Christophe agrees, I've updated it :+1: